lib/hooks/search-users.ts

```diff
@@ -1,21 +1,9 @@
-const searchUsers = async (username: string, providerToken?: string | null | undefined) => {
+const searchUsers = async (username: string) => {
```

Comment:
This needs to be reverted. We switched to use the GitHub search API for more flexibility

#2331